new-log-viewer/src/services/LogExportManager.ts

```diff
@@ -0,0 +1,42 @@
+import {downloadBlob} from "../utils/file";
+
+
+class LogExportManager {
```

Comment:
Let's write JSDoc for all public methods in this class. Ideally, docs should also be written for private methods.